### ISSUE TITLE
Export Button

### ISSUE THREAD

**Issue by domoritz** [vega/editor#5]
From https://github.com/vega/vega-editor/issues/184

**Comment by domoritz** [comment]
The export button/print mode should just show the plot so you can print it.

**Comment by ishansaksena** [comment]
https://github.com/vega/vega-embed

**Comment by domoritz** [comment]
I don't think so. When you view a chart in full screen you can also save or print as PDF.

**Comment by saurabh0402** [comment]
@domoritz If it's fine with you I would like to work on this.

**Comment by domoritz** [comment]
I know that @ishansaksena already started working on this. He might have some starter code.

**Comment by ydlamba** [comment]
Working on export button.

**Comment by domoritz** [comment]
🎉

**Comment by domoritz** [comment]
Do you have a suggestion where to put the export button? I feel out toolbar in the bottom right is overfowing. Also, we have a parse button above the editor in manual mode. Later, we also want to place the format button (https://github.com/vega/editor/issues/37) somewhere.

**Comment by ydlamba** [comment]
I think all actionable buttons should be together. So that we can refer them as a toolbar.
![image](https://user-images.githubusercontent.com/21237466/37012644-bd0d7558-211b-11e8-8a43-2b6abd70ff79.png)

**Comment by domoritz** [comment]
As I said, another option is right above the editor. I think where you have it is good for now.
